subsys/bluetooth/audio/shell/audio.h

```diff
@@ -166,6 +189,66 @@ static inline void print_base(const struct shell *sh, const struct bt_bap_base *
 }
 #endif /* BROADCAST_SNK_SUBGROUP_CNT > 0 */
 
+static inline void copy_unicast_stream_preset(struct unicast_stream *stream,
+					      const struct named_lc3_preset *named_preset)
+{
+	memcpy(&stream->qos, &named_preset->preset.qos, sizeof(stream->qos));
```

Comment:
(This might be a stupid question) Why aren't we using (void) before memcpy here?

Reply:
It's something I'd usually do, and recommend as well, but from several discussions with people both in and outside the BT subsystem, it seems that I am the minority in wanting this, and thus I stopped casting it to `void` :) Until we have a coding guideline that says otherwise, any way of doing it is fine.

Reply:
>  I am the minority in wanting this

If am right original kernel and public interface code did/does use. And hence recently added Controller code does add it. 

That said, I have no strong request to change in this PR.

Reply:
We have 327 uses of `(void)memcpy` and 2045 uses of `memcpy` in Zephyr, so it is by far most common to use it without `(void)`. I did "enforce" it in my reviews for a while, as I do believe it's the right approach, but it doesn't matter if it is not universally used or expected by the coding guidelines :(

Reply:
Ok thanks for the explanation, I'm fine with this as it is right now :)